.build.config.js

```diff
@@ -1,31 +0,0 @@
-var srcFolder = "./src";
```

Comment:
I did `git mv` on all moved files and don't understand why it hasn't been honored..